bin/bootstrap.sh

```diff
@@ -5,9 +5,9 @@ ENV_DIR=${ENV_DIR:-./env}
 . bin/common.sh
 
 . $ENV_DIR/.secrets
-
-skip_demo_files=$1
-[ -f $ENV_DIR/bin/otomi ] && has_otomi=true
+has_otomi=0
```

Comment:
Why not use undefined that is available in bash, so you can skip this line?

Reply:
the `-u` flag forces all variables to be explicitly defined even if they fallback to undefined.

Reply:
having undefined variable is also insecure because they can be overwritten by environment variables

Reply:
Ok, then use true|false when boolean is intended.

Reply:
just to make it clear in BASH there are no `true` or `false` values that you can assign.
The true and false that you have used are commands: eg.g: /usr/bin/true

Reply:
we can make a convention to use strings instead

Reply:
True|false are there for a reason.

Reply:
So do you want to use strings "false" and "true" instead of "0" and "1"?

Reply:
done